dev-docs/bidders/flatads.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: bidder
+title: flatads
+description: Prebid Flatads Bidder Adapter
+biddercode: flatads
+tcfeu_supported: false
+usp_supported: true
+coppa_supported: true
+userIds: all
+schain_supported: true
+dchain_supported: true
+media_types: banner, video, native
+safeframes_ok: false
+deals_supported: true
+floors_supported: true
+pbjs: false
+pbs: true
+pbs_app_supported: true
+prebid_member: false
+multiformat_supported: will-bid-on-any
+ortb_blocking_supported: true
+sidebarType: 1
+---
+
+### Registration
+
+Flatads header bidding adapter connects with Flatads demand sources to fetch bids for banner publisher ID. Please reach
+out to your account manager or <tangyang@flat-ads.com> for more information.
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name | Scope | Description | Example | Type |
+|---------------|----------|--------------|-----------|----------|
+| `publisherId` | required | The publisher ID from Flatads | `1111` | `string` |
```

Comment:
Please add quotes around both of these string params. The backticks resolve down to formatting, so the quotes are missing.